app/views/courses/_users.json.jbuilder

```diff
@@ -4,7 +4,7 @@ show_email_and_real_name = user_signed_in? && current_user.role(course).positive
 
 json.users course.courses_users.eager_load(:user, :course) do |cu|
   json.call(cu, :character_sum_ms, :character_sum_us, :character_sum_draft, :role,
-            :recent_revisions, :content_expert, :program_manager, :contribution_url, :sandbox_url)
+            :recent_revisions, :content_expert, :program_manager, :contribution_url, :sandbox_url, :real_name)
```

Comment:
Instead of adding it here, the call to cu.real_name should replace `cu.user.real_name` on line 20.